src/classes/STG_PanelOppBatch_CTRL.cls

```diff
@@ -59,9 +62,34 @@ public with sharing class STG_PanelOppBatch_CTRL extends STG_Panel {
     public PageReference runBatch() {
         isRunningBatch = true;
 
-        // run batch rollup
-        RLLP_OppRollup rg = new RLLP_OppRollup();
-        rg.rollupAll();        
+        if (!CRLP_Rollup_SVC.isCustomizableRollupEngineEnabled) {
+            // run legacy batch rollups
+            RLLP_OppRollup rg = new RLLP_OppRollup();
+            rg.rollupAll();
+        } else {
+            Integer batchSizeAcct = CRLP_Rollup_SVC.getBatchSize(Account.SObjectType);
+            Integer batchSizeCntct = CRLP_Rollup_SVC.getBatchSize(Contact.SObjectType);
+
+            Database.executeBatch(new CRLP_Rollup_BATCH(Account.SObjectType), batchSizeAcct);
+            Database.executeBatch(new CRLP_Rollup_BATCH(Contact.SObjectType), batchSizeCntct);
+            this.numJobSubmitted = 2;
+
+            Integer maxRelatedOppsForNonLDVMode = CRLP_Rollup_SVC.getMaxRelatedOppsForNonLDVMode();
+            Boolean hasLDVAccounts = [SELECT Count() FROM Account WHERE Number_of_Opportunities__c >= :maxRelatedOppsForNonLDVMode LIMIT 1] > 0;
```

Comment:
I've seen this a couple times, could be a method somewhere.

Reply:
The condition itself is included a few times, but I was unable to find a second query like this that is against the entire Account object